build/webpack/webpack.client.config.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
         renderers: './src/client/index.tsx'
     },
     output: {
-        path: path.join(constants.ExtensionRootDir, 'out', 'client'),
+        path: path.join(constants.ExtensionRootDir, 'out', 'client_renderer'),
```

Comment:
Changed target directories so they do not conflict with `client` & `extension` folders in the Jupyter extension. Suffixed with `_renderer` so that we can copy them blindly across to Juypyter extension output directory & not worry about fixing paths.